summingbird-online/src/main/scala/com/twitter/summingbird/planner/OnlinePlan.scala

```diff
@@ -27,6 +29,17 @@ class OnlinePlan[P <: Platform[P], V](tail: Producer[P, V]) {
   private val depData = Dependants(tail)
   private val forkedNodes = depData.nodes
     .filter(depData.fanOut(_).exists(_ > 1)).toSet
+
+  private def get[T <: AnyRef: ClassTag](dep: Producer[P, _], nameMap: Map[Producer[P, _], List[String]]): Option[(String, T)] = {
```

Comment:
why do you need to pass the nameMap here since it is a class variable? Also below?